smoketests/tests/permissions.py

```diff
@@ -21,28 +19,23 @@ def test_call(self):
     def test_delete(self):
         """Ensure that you cannot delete a database that you do not own"""
 
-        self.new_identity()
```

Comment:
I don't believe this `new_identity` really added anything to the test, but it hamstrung the test when running against servers that required a `spacetime login`.